extensions/sshfs/src/components/ActiveMountList.tsx

```diff
@@ -0,0 +1,54 @@
+import { List, ActionPanel, Action, useNavigation } from "@raycast/api";
+import { ActiveMount } from "../types";
+import { unmountPath } from "../utils/mount";
+
+interface ActiveMountListProps {
+  activeMounts: ActiveMount[];
+  onRefresh: () => void;
+  language: string;
+}
+
+export function ActiveMountList({ activeMounts, onRefresh, language }: ActiveMountListProps) {
+  const { pop } = useNavigation();
+  const handleUnmount = async (mount: ActiveMount) => {
+    try {
+      await unmountPath(mount.mountPoint);
+      await onRefresh();
+      pop();
+    } catch {
+      // 에러는 unmountPath 함수에서 처리됨
+    }
+  };
+
+  return (
+    <List searchBarPlaceholder={language === "ko" ? "활성 마운트 검색..." : "Search active mounts..."}>
+      {activeMounts.length === 0 ? (
+        <List.EmptyView
+          title={language === "ko" ? "활성 마운트가 없습니다" : "No active mounts"}
+          description={language === "ko" ? "현재 마운트된 SSHFS가 없습니다" : "No mounted SSHFS"}
+        />
```

Comment:
See https://developers.raycast.com/basics/prepare-an-extension-for-store#localization-language.

At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.